src/scenarioFilter.js

```diff
@@ -0,0 +1,21 @@
+import logger from './logger';
+
+const filterScenario = (config, filter) => {
+  return new Promise(resolve => {
```

Comment:
mmmmm i think we might able to do this a bit nicer

i'll come sit

Reply:
mm you weren't there  😢

I don't think we are doing anything async here so could remove the promise all together.

should be able to do something like
```
 if (options.run) {
    config.scenarios =  filterScenarios(config, options.run)
}
```